addons/repair/migrations/13.0.1.0/post-migration.py

```diff
@@ -0,0 +1,66 @@
+# Copyright 2021 Tecnativa - Víctor Martínez
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+from psycopg2 import sql
+from openupgradelib import openupgrade
+
+
+def update_repair_fee_invoice_relation(env):
+    openupgrade.logged_query(
+        env.cr, sql.SQL(
+            """UPDATE repair_fee rf
+            SET invoice_line_id = aml.id
+            FROM account_invoice_line ail
+            JOIN account_move_line aml ON aml.old_invoice_line_id = ail.id
+            WHERE rf.{} = ail.id"""
```

Comment:
Extra table linked with no need:
```suggestion
            """UPDATE repair_fee rf
            SET invoice_line_id = aml.id
            FROM account_move_line aml
            WHERE rf.{} = aml.old_invoice_line_id"""
```
and the rest of the queries applies the same